### PR TITLE
LPS-126647

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LPS-126647
@david-gutierrez-mesa
@liulukiwi
@Tim-Cao

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

**Comment by nkkwon** [comment]
ci:test:relevant

**Comment by nkkwon** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase [hunk 1]
```diff
@@ -250,8 +250,33 @@ definition {
...
+		WebContentStructures.saveCP(structureName = "WC Structure Name");
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+		NavItem.gotoTemplates();
+
+		WebContentTemplates.addCP(
+			structureName = "WC Structure Name",
+			templateDescription = "WC Template Description",
+			templateFieldNames = "Text",
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase:272]: Should add other field values in template except Separator

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase [hunk 2]
```diff
@@ -250,8 +250,33 @@ definition {
...
+			templateName = "WC Template Name");
+
+		WebContentTemplates.modifyTemplate(
+			templateName = "WC Template Name Edit",
+			templatePreviousName = "WC Template Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase:277]: Add a step to validate template is renamed

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase [hunk 3]
```diff
@@ -250,8 +250,33 @@ definition {
...
 
+		WebContentTemplates.deleteCP(templateName = "WC Template Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase:279]: Add a step to validate template is deleted

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase [hunk 4]
```diff
@@ -250,8 +250,33 @@ definition {
 	test ViewAndEditAsWellAsDeleteTemplateWithAllFieldsStrucrure {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase:250]: Rename to  ViewAndEditAsWellAsDeleteTemplateWithAllFieldsStructure

**Comment by Tim-Cao** [comment]
Hi @nkkwon, the verb should be present tense in commit message.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase [hunk 5]
```diff
@@ -250,8 +250,34 @@ definition {
...
+			templateDescription = "WC Template Description",
+			templateFieldNames = "Text",
+			templateName = "WC Template Name");
+
+		WebContentTemplates.modifyTemplate(
+			templateName = "WC Template Name Edit",
+			templatePreviousName = "WC Template Name");
+
 
+		WebContentTemplates.deleteCP(templateName = "WC Template Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/template/WebContentTemplates.testcase:279]: WC Template Name is renamed to WC Template Name Edit

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3564
View total diff https://github.com/liferay-echo/liferay-portal/pull/3564/commits/a8e0280e9acf15cbc91d494c414a66dfa788e051

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**